app/graphql/types/query_types/survey_poll_type.rb

```diff
@@ -8,6 +8,8 @@ class QueryTypes::SurveyPollType < Types::BaseObject
     field :description, QueryTypes::I18nJSON, null: true
     field :date, QueryTypes::DateType, null: true
     field :response_options, [QueryTypes::SurveyResponseOptionsType], null: true
+    field :data_provider, QueryTypes::DataProviderType, null: true
+    field :visible, Boolean, null: true
```

Comment:
as in the other PR: what is this for?

Reply:
answered there: https://github.com/ikuseiGmbH/smart-village-app-cms/pull/70#discussion_r653677717